pyunlocbox/tests/test_operators.py

```diff
@@ -20,45 +20,46 @@ def test_grad(self):
         mat2d = np.array([[2, 3, 0, 1], [22, 1, 4, 5]])
         mat3d = np.arange(1, 13).reshape(2, 2, 3).transpose((1, 2, 0))
         mat4d = np.arange(1, 25).reshape(2, 2, 2, 3).transpose((2, 3, 1, 0))
-        mat5d = np.arange(1, 49).reshape(2, 2, 3, 2, 2).transpose((3, 4, 2, 1, 0))
+        mat5d = np.arange(1, 49).reshape(2, 2, 3, 2, 2)
+        mat5d = mat5d.transpose((3, 4, 2, 1, 0))
 
         # 1D without weights.
-        dx = operators.grad(mat1d, dim=1)
+        dx = operators.grad1d(mat1d)
```

Comment:
This was just to improve coverage in the tests. The operators module defines functions grad1d(x), grad2d(x), etc. which are just wrappers for calling grad(x, dim=1), grad(x, dim=2), etc., respectively. Calling grad with the dim argument explicitly in the test script was biasing the coverage down, essentially because grad1d(x), grad2d(x), etc. were never being called.